.github/workflows/publish.yml

```diff
@@ -69,5 +69,3 @@ jobs:
         with:
           branch: gh-pages
           folder: output
-        secrets:
-          token: ${{ secrets.GH_TOKEN }}
```

Comment:
As far as I understood the documentation there is no need to provide the token here either, is there @jesserockz?
https://github.com/JamesIves/github-pages-deploy-action#required-setup